pkg/controller/backup_controller.go

```diff
@@ -224,6 +227,7 @@ func (controller *backupController) processBackup(key string) error {
 
 	logContext.Debug("Cloning backup")
 	// don't modify items in the cache
+	original := *backup
```

Comment:
Why'd you deref the pointer?

Reply:
This also makes it look like this is a safe way to avoid modifying items in the cache